schematic/models/validate_attribute.py

```diff
@@ -1577,7 +1712,25 @@ def _run_validation_across_target_manifests(
         val_rule: str,
         manifest_col: pd.core.series.Series,
         target_column: pd.core.series.Series,
-    ) -> tuple[[float, list]]:
+    ) -> tuple[
+        float,
+        Union[
```

Comment:
The outer `Union` here is redundant as it only contains the inner `Union`

Reply:
Okay its fixed, It should have two unions, but I accidentally put bool and str within the second union.